systemtest/src/test/java/io/strimzi/systemtest/upgrade/kraft/AbstractKRaftUpgradeST.java

```diff
@@ -77,7 +76,7 @@ protected void deployKafkaClusterWithWaitForReadiness(final ExtensionContext ext
                 resourceManager.createResourceWithWait(extensionContext,
                     KafkaNodePoolTemplates.kafkaNodePoolWithControllerRoleAndPersistentStorage(TestConstants.CO_NAMESPACE, CONTROLLER_NODE_NAME, clusterName, 3).build(),
                     KafkaNodePoolTemplates.kafkaNodePoolWithBrokerRoleAndPersistentStorage(TestConstants.CO_NAMESPACE, BROKER_NODE_NAME, clusterName, 3).build(),
-                    KafkaTemplates.kafkaPersistent(clusterName, 3, 3)
+                    KafkaTemplates.kafkaPersistentKRaft(clusterName, 3)
```

Comment:
isn't it also used in non-kraft tests? If so, the name is a little bit misleading

Reply:
FMPOV we should have different templates for KRaft and non-KRaft Kafka CRs and depending on FG env variable pick one or another. I need to have a template that doesn't contain LMFV and IBPV, otherwise it will fail the Kafka containers because of wrong version of those set during the downgrade. If the name is confusing - what would you pick?

Reply:
Discussed offline, lgtm